Source: oahong@gmail.com
Upstream: mail sent
Reason: Fix compilation problem due to gcc 4.4 header cleanups
--- openvanilla-modules-0.8.0_9/OVIMGeneric/OVIMGeneric.cpp.orig	2009-05-08 02:04:58.075634949 +0800
+++ openvanilla-modules-0.8.0_9/OVIMGeneric/OVIMGeneric.cpp	2009-05-08 02:04:46.533809062 +0800
@@ -30,6 +30,7 @@
 
 // #define OV_DEBUG
 
+#include <stdio.h>
 #include "OVIMGeneric.h"
 #include "CIN-Defaults.h"
 
--- openvanilla-modules-0.8.0_9/SharedSource/OVCINInfo.cpp.orig	2009-05-08 02:25:04.591885368 +0800
+++ openvanilla-modules-0.8.0_9/SharedSource/OVCINInfo.cpp	2009-05-08 02:24:26.984885996 +0800
@@ -31,6 +31,7 @@
 // #define OV_DEBUG
 
 #include "OVCINInfo.h"
+#include <stdio.h>
 
 #ifndef WIN32
     #include <OpenVanilla/OpenVanilla.h>
--- openvanilla-modules-0.8.0_9/SharedSource/OVCandidateList.cpp.orig	2009-05-08 02:28:54.486635334 +0800
+++ openvanilla-modules-0.8.0_9/SharedSource/OVCandidateList.cpp	2009-05-08 02:29:23.608634394 +0800
@@ -30,6 +30,7 @@
 
 #include "OVCandidateList.h"
 #include "OVCIN.h"
+#include <stdio.h>
 #include <iostream>
 
 #ifdef __linux__
